.github/workflows/ci-actions.yml

```diff
@@ -130,6 +154,11 @@ jobs:
           sudo service mysql stop || true
 
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Add quarkusio remote
+        shell: bash
+        run: git remote add quarkusio https://github.com/quarkusio/quarkus.git
```

Comment:
Notes:
- this entire block is now present for almost each job
- I decided to only add the remote, not directly fetch from it, so that GIB can decide whether or not to fetch anything at all (see the "Get GIB arguments" step further up for the cases where GIB disables itself)